docroot/profiles/humsci/su_humsci_profile/js/select-lists/src/select-list.tsx

```diff
@@ -268,11 +268,7 @@ const SelectList = ({
       {label && (
         <div
           id={labelId}
-          style={{
-            marginBottom: '0.75rem',
-            fontSize: '1.125rem',
-            fontWeight: '600',
-          }}
+          class="preact-label"
```

Comment:
Can you do this class more specific? preact is the name of the library used for this and there's a chance it could be used for something else in the future. Parent element uses `select-preact`, so let's use  `select-preact__label`

```suggestion
          class="select-preact__label"
```